### PR TITLE
add coverage export empty box

### PR DESCRIPTION
**Pull request by nberard**

### CODE REVIEW

File: tartare/core/coverage_export_functions.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import logging
+from tartare.core.models import ContributorExport
+
+logger = logging.getLogger(__name__)
+
+
+def merge(coverage, context):
+    logger.info("coverage_id : %s", coverage.id)
+
```
Comment (azime) [inline tartare/core/coverage_export_functions.py]: return context

File: tartare/core/coverage_export_functions.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+def postprocess(coverage, context):
+    logger.info("coverage_id : %s", coverage.id)
+
```
Comment (azime) [inline tartare/core/coverage_export_functions.py]: return context

File: tartare/core/models.py [hunk 3]
```diff
@@ -467,7 +468,7 @@ def update(cls, job_id, state=None, step=None, error_message=None):
 class MongoJobSchema(Schema):
     id = fields.String(required=True, load_from='_id', dump_to='_id')
     action_type = fields.String(required=True)
-    contributor_id = fields.String(required=True)
+    contributor_id = fields.String(required=False)
```
Comment (azime) [inline tartare/core/models.py]: coverage_id = fields.String(required=False)

Comment (nberard) [inline tartare/core/models.py]: nice catch :)

**Pull request merged**